Content.Server/Battery/DrainAllBatteriesCommand.cs

```diff
@@ -24,10 +24,11 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
             }
 
             var entityManager = IoCManager.Resolve<IEntityManager>();
-            foreach (var ent in entityManager.GetEntities(new TypeEntityQuery(typeof(BatteryComponent))))
+            foreach (BatteryComponent batteryComp in entityManager.ComponentManager.GetAllComponents(typeof(BatteryComponent)))
```

Comment:
This is doing an implicitly casted loop, you want this instead:
```suggestion
            foreach (var batteryComp in entityManager.ComponentManager.EntityQuery<BatteryComponent>())
```

(Basically they decided to bandaid the lack of generics in C# 1 by allowing you to implicitly cast the loop variable, it's really bad and mostly a code wart nowadays...)